client/flutter/lib/pages/onboarding/legal_landing_page.dart

```diff
@@ -38,12 +38,20 @@ class LegalLandingPage extends StatelessWidget {
                 style: TextStyle(color: Colors.grey),
                 children: [
                   TextSpan(
-                    text: "By proceeding, you agree with our "
+                    text: "By proceeding, you agree to our "
                   ),
                   LinkTextSpan(
-                    text: "terms of service",
+                    text: "Terms of Service",
                     style: TextStyle(decoration: TextDecoration.underline),
-                    url: "https://who.int",//TODO: REPLACE WITH TERMS OF SERVICE LINK
+                    url: "https://whocoronavirus.org/terms",
```

Comment:
I wasn't sure if @crazybob said this or https://whocoronavirus.org/tos on the phone just now... I'm sure it can be set up either way

Reply:
it doesn't matter lol